mix.exs

```diff
@@ -37,9 +37,9 @@ defmodule EctoHLClock.Mixfile do
     [
       {:ecto, "~> 3.0"},
       {:ecto_sql, "~> 3.0"},
-      {:hlclock, "~> 1.0"},
+      {:hlcid, "~> 0.1"},
```

Comment:
I'm not sure if we want to depend on hlcid here instead of hlclock. hlcid pulls in the entire application and that may not be required in all use cases. Is there a conflict in the dependencies if we don't do it this way?

Reply:
Oh, its for autogeneration. I get it.